x-pack/plugins/ml/public/data_frame/components/job_create/job_create_form.tsx

```diff
@@ -162,52 +180,203 @@ export const JobCreateForm: SFC<Props> = React.memo(
-              defaultMessage: 'Start data frame',
-            })}
-          </EuiButton>
+          <EuiFlexGroup alignItems="center" style={{ maxWidth: '800px' }}>
+            <EuiFlexItem grow={false} style={{ width: '200px' }}>
+              <EuiButton isDisabled={created && started} onClick={startDataFrame}>
+                {i18n.translate('xpack.ml.dataframe.jobCreateForm.startDataFrameButton', {
+                  defaultMessage: 'Start',
+                })}
+              </EuiButton>
+            </EuiFlexItem>
+            <EuiFlexItem>
+              <EuiText color="subdued" size="s">
+                {i18n.translate('xpack.ml.dataframe.jobCreateForm.startDataFrameDescription', {
+                  defaultMessage:
+                    'Starts the data frame job. After the job is started, you will be offered options to continue exploring the data frame job.',
+                })}
+              </EuiText>
+            </EuiFlexItem>
+          </EuiFlexGroup>
         )}
-        {created && started && (
+        <EuiFlexGroup alignItems="center" style={{ maxWidth: '800px' }}>
+          <EuiFlexItem grow={false} style={{ width: '200px' }}>
+            <EuiButton isDisabled={created} onClick={createDataFrame}>
+              {i18n.translate('xpack.ml.dataframe.jobCreateForm.createDataFrameButton', {
+                defaultMessage: 'Create',
+              })}
+            </EuiButton>
+          </EuiFlexItem>
+          <EuiFlexItem>
+            <EuiText color="subdued" size="s">
+              {i18n.translate('xpack.ml.dataframe.jobCreateForm.createDataFrameDescription', {
+                defaultMessage:
+                  'Create the data frame job without starting it. You will be able to start the job later by returning to the data frame jobs list.',
+              })}
+            </EuiText>
+          </EuiFlexItem>
+        </EuiFlexGroup>
+        <EuiFlexGroup alignItems="center" style={{ maxWidth: '800px' }}>
+          <EuiFlexItem grow={false} style={{ width: '200px' }}>
+            <EuiCopy textToCopy={getJobConfigDevConsoleStatement()}>
+              {(copy: () => void) => (
+                <EuiButton onClick={copy} style={{ width: '100%' }}>
+                  {i18n.translate(
+                    'xpack.ml.dataframe.jobCreateForm.copyJobConfigToClipBoardButton',
+                    {
+                      defaultMessage: 'Copy to clipboard',
+                    }
+                  )}
+                </EuiButton>
+              )}
+            </EuiCopy>
+          </EuiFlexItem>
+          <EuiFlexItem>
+            <EuiText color="subdued" size="s">
+              {i18n.translate(
+                'xpack.ml.dataframe.jobCreateForm.copyJobConfigToClipBoardDescription',
+                {
+                  defaultMessage:
+                    'Copies a Kibana Dev Console statement to create the job to the clipboard.',
```

Comment:
Tempted to reword this slightly to `Copies to the clipboard the Kibana Dev Console statement for creating the job.` Maybe use `command` rather than `statement`?

Reply:
Fixed in 121f2c9.